### PR TITLE
Adds Surgical toolarms to R&D, makes them less robust

### PR DESCRIPTION
**Pull request by Anonmare**
Also removes the W_CLASS override from arm augments, it didn't appear to be doing anything at any rate.

:cl:
add: Surgical toolarm to protolathe and exofab
tweak: Toolarm tools are less robust but more efficient at surgery
tweak: Arm augments are no longer a huge item
/:cl:

[why]: Same as before. It doesn't feel good for the surgical toolarm to be left to languish, and keeps the tools no more effective than a kitchen knife in terms of damage.

Toolarm tools do no more than 10 force damage and do not exceed W_CLASS SMALL. This will reduce their chances of knockdown and their actual effectiveness in combat. I hope everyone feels this version is less powercreep-y.

### CODE REVIEW

**Comment by RemieRichards** [comment]
There was no need for a second PR.

**Comment by Anonmare** [comment]
Don't know how to add additional files to a PR. Tried asking but wasn't getting a response in the Discord

**Comment by RemieRichards** [comment]
>The Discord

there's your problem, you want help come to coderbus directly.

File: augments_arms.dm [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+		emag_act()
+
+/obj/item/organ/cyberimp/arm/Remove(mob/living/carbon/M, special = 0)
+	Retract()
+	..()
+
+/obj/item/organ/cyberimp/arm/emag_act()
+	return 0
+
+/obj/item/organ/cyberimp/arm/gun/emp_act(severity)
```
Comment (octareenroon91) [inline augments_arms.dm]: possible copypaste error

**Comment by octareenroon91** [comment]
I see new files being created, but no reference to them in `tgstation.dme`. Moving existing code to new files is presumably fine, but it's got to be a move not a copy.

**Comment by kevinz000** [comment]
your pr is broken....

File: medical_designs.dm [hunk 2]
```diff
@@ -0,0 +1,401 @@
+/////////////////////////////////////////
+////////////Medical Tools////////////////
```
Comment (ChangelingRain) [inline medical_designs.dm]: you gotta remove the old designs from wherever they are

Comment (Anonmare) [inline medical_designs.dm]: What do you mean by that?

Comment (Chronitonity) [inline medical_designs.dm]: You made a new medical_designs.dm for some reason. 

[See?](http://i.imgur.com/mq6GnQD.png)

**Comment by Anonmare** [comment]
Alrighty I finally figured Git out, hopefully this should fix the problems

File: code/modules/surgery/tools.dm [hunk 3]
```diff
@@ -141,4 +232,4 @@
 		desc = "A container for holding body parts."
 	else
 		to_chat(user, "[src] is empty.")
-	return
+return
```
Comment (AnturK) [inline code/modules/surgery/tools.dm]: Same here.

File: code/modules/surgery/organs/augments_arms.dm [hunk 4]
```diff
@@ -234,5 +233,5 @@
 /obj/item/organ/cyberimp/arm/surgery
 	name = "surgical toolset implant"
 	desc = "A set of surgical tools hidden behind a concealed panel on the user's arm"
-	contents = newlist(/obj/item/weapon/retractor, /obj/item/weapon/hemostat, /obj/item/weapon/cautery, /obj/item/weapon/surgicaldrill, /obj/item/weapon/scalpel, /obj/item/weapon/circular_saw, /obj/item/weapon/surgical_drapes)
-	origin_tech = "materials=3;engineering=3;biotech=3;programming=2;magnets=3"
+	contents = newlist(/obj/item/weapon/retractor/augment, /obj/item/weapon/hemostat/augment, /obj/item/weapon/cautery/augment, /obj/item/weapon/surgicaldrill/augment, /obj/item/weapon/scalpel/augment, /obj/item/weapon/circular_saw/augment, /obj/item/weapon/surgical_drapes)
+origin_tech = "materials=3;engineering=3;biotech=3;programming=2;magnets=3"
```
Comment (AnturK) [inline code/modules/surgery/organs/augments_arms.dm]: You dropped indentation here.

**Comment by Anonmare** [comment]
@KorPhaeron

**Comment by Repukan** [comment]
Don't tool arms accomplish this already due to ghetto surgery?

**Comment by octareenroon91** [comment]
>Don't tool arms accomplish this already due to ghetto surgery?

Except for drapes, success rate, and maybe speed, I suppose so.

This PR is kind of a mess.

**Comment by KorPhaeron** [comment]
I dont care about this either way

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**